src/fidesops/service/connectors/saas_connector.py

```diff
@@ -279,6 +286,40 @@ def process_response_data(
 
         return rows
 
+    def get_masking_request_from_config(
+        self, collection_name: str
+    ) -> Optional[SaaSRequest]:
+        """Get the configured SaaSRequest for use in masking.
+        An update request is preferred, but we can use a gdpr delete endpoint or delete endpoint if not MASKING_STRICT.
+        """
+        requests = self.endpoints[collection_name].requests
+
+        update: Optional[SaaSRequest] = requests.get("update")
+        gdpr_delete: Optional[SaaSRequest] = None
+        delete: Optional[SaaSRequest] = None
+
+        if not config.execution.MASKING_STRICT:
```

Comment:
nice, I like the way you set this up so that delete isn't even a populated option when we're in strict mode.

Reply:
thank you @eastandwestwind!